packages/forma-36-react-components/src/components/Card/CardActions/CardActions.tsx

```diff
@@ -55,7 +55,6 @@ export class CardActions extends Component<
     const {
       className,
       children,
-      testId,
```

Comment:
@arpl I don't think this should be removed?

Reply:
Instead, we should explicitly pass it to the `Dropdown` component below.

Reply:
Since `CardAction.testId` matches `Dropdown.testId` and it felt 'too' explicit ;)

Reply:
```
    const {
      className,
      children,
      testId,
      iconButtonProps,
      isDisabled,
      ...otherProps
    } = this.props;

    return (
      <Dropdown
        onClose={() => {
          this.setState({
            isDropdownOpen: false,
          });
        }}
        position="bottom-right"
        className={className}
        isOpen={this.state.isDropdownOpen}
        testId={testId}
        toggleElement={
          <IconButton
            iconProps={{ icon: 'MoreHorizontal' }}
            buttonType="secondary"
            disabled={isDisabled}
            label="Actions"
            {...iconButtonProps}
            onClick={event => {
              event.preventDefault();
              this.handleClick(event);
            }}
          />
        }
        {...otherProps}
      >

```